src/ifp/include/ifp/InitFloorplan.hh

```diff
@@ -120,6 +131,10 @@ class InitFloorplan
   dbBlock* block_;
   Logger* logger_;
   sta::dbNetwork* network_;
+
+ private:
```

Comment:
Just change the whole protected section to private rather than make this one field private.  Nobody is inheriting from this class.